### DISCUSSION TITLE
Impossible de se connecter à mon compte avec abonnement payant

### DISCUSSION THREAD

**Discussion by Alex06400** [AmauriC/tarteaucitron.js#887] [General]
Bonjour,

Je n’arrive plus à me connecter à mon compte, j’ai « identifiants invalides » ou un message « ERROR » si je fais mot de passe perdu.

uuid: fd65160674195481416fff28ae48c37fee93a38d

Existe-t-il une option pour récupérer mon compte car je dois renouveler l'abonnement prochainement ?

Merci !

**Comment by AmauriC** [comment]
Hello

Le système de changement de mot de passe n'est pas banal.
Il faut entrer votre email + votre nouveau mot de passe et cocher la case "mot de passe perdu" sur https://tarteaucitron.io/fr/pro/login/

En validant vous recevez un mail, en cliquant sur le lien du mail, cela active le nouveau mdp.

**Comment by Alex06400** [comment]
Hello,

Ca a fonctionné, merci beaucoup, j'ai "récupére" mon compte  ;-)
